index.html

```diff
@@ -1,16 +1,90 @@
+<!-- Responsive Web Design -->
 <!doctype html>
 
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
+  
 </head>
 
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+  <body>
+    <section>
+<div>
+  <nav> 
+    <ul>
+      <li><a href =“#”>Services</a></li>
+      <li><a href= “#”>Product</a></li>
+      <li><a href=“#”>Vision</a></li>
+      <li><a href=“#”>Features</a></li>
+      <li><a href=“#”>About</a></li>
+      <li><a href=“#”>Contact</a></li> 
+      <li><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></li>     
+     </ul>
+     
+  </nav> 
+</div>
+<div class="divTop"> 
+      <span class="a"><h2>Innovation<br> 
+        On<br>
+        Demand<br><a class="gStarted" href="#">Get Started</a></h2></span><span class="b"><img class="img1" src="img/header-img.png" alt="Image of a code snippet."></span>
+      
+       
+
+
+</div> 
+
+</section>
+
+ <section>
+ 
+  <div class="divMiddle">
+   <span class="c"> 
+    <h1>Features</h1> <br>
+   <p class="p1"> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</span> 
+<span class="d">
+    <h1>About</h1><br>
+    <p class="p1">Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</span><br><br>
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <span class="e">
+    <h1>Services</h1><br>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</span>
+    <span class="f">
+    <h1>Product</h1><br>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</span>
+    <span class="f">
+    <h1>Vision</h1><br>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</span> <br> <br>
+</div>
+</section> 
+
+<section>  
+    
+  <div class="divBottom">
+  <span class="g">
```

Comment:
Naming meaningful classes is important for you and other developers, because when your codebase goes large, it is hard to recognize a `g` class in the sea of css styles 😅